### PR TITLE
Created glossary.md and defined project brand related terms

### LINKED ISSUE #323: Describe the difference between notary, TUF, Notary Project specification, and notation in faq.md
Describe what is what, what it means, how it works, and how it is different from the others.

### PR DESCRIPTION
**Pull request by Roseline-Bassey**
This PR includes the following:
- A new glossary.md page 
-  It described some Notary Project terminologies such as Notary, Notation, Notary Project Specifications, and TUF.

This PR fixes #323 

The difference between these terminologies would be documented in FAQ through another PR.

### CODE REVIEW

File: content/en/docs/glossary.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- ### Notation
+Notation is a non-TUF-based artifact signing solution. It refers to the implementation of the Notation CLI and the Notation Go libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with 3rd party key management solutions through a plugin model. The CLI itself is referred to as the "Notation CLI," while the libraries are referrred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. These libraries can be used independently of the Notation CLI.
+
```
Comment (yizha1) [inline content/en/docs/glossary.md:12]: Remove extra line

File: content/en/docs/glossary.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- [Notary Project OCI Signature Specification](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md)
+- [Notary Project OCI Signing and Verification Workflow](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-and-verification-workflow.md)
+- [Notary Project OCI COSE Envelope](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-cose.md)
+- [Notary Project OCI JWS Envelope](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-jws.md)
+
+**Signing Scheme Specification:** The Signing Scheme Specification is not specific to any particular storage method. It describes the supported signing schemes. Any tool that supports those schemes might be able to produce signatures that are compatible with Notary Project tools. The specification under this category includes the [Notary Project Signing Scheme.](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-scheme.md)
+
```
Comment (yizha1) [inline content/en/docs/glossary.md:27]: Remove extra line

File: content/en/docs/glossary.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- ### TUF
+The Update Framework(TUF) is a project under the Notary Project that aims to implement the full TUF specification in a registry-native way. The initial version of Notary did not support transferring signatures between different registries and did not gain widespread adoption. The TUF project aims to develop an enhanced version of Notary, addressing these limitations and encouraging widespread adoption and usage.
+
```
Comment (yizha1) [inline content/en/docs/glossary.md:31]: Remove extra line

File: content/en/docs/glossary.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+title: Glossary
+description: A list of Notary Project's terminologies 
+weight: 11
+---
+
+- ### Notary
```
Comment (yizha1) [inline content/en/docs/glossary.md:7]: Remove `-`

File: content/en/docs/glossary.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
+title: Glossary
+description: A list of Notary Project's terminologies 
+weight: 11
+---
+
+- ### Notary 
+Notary is a TUF [(The Update Framework)](https://github.com/theupdateframework/specification) based artifact signing solution. It consists of both [server](https://github.com/notaryproject/notary/blob/master/cmd/notary-server) and [client](https://github.com/notaryproject/notary/blob/master/cmd/notary) components for running and interacting with trusted collections. The client component is referred to as the Notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component.
+
+- ### Notation
```
Comment (yizha1) [inline content/en/docs/glossary.md:10]: Remove `-`

File: content/en/docs/glossary.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+Notation is a non-TUF-based artifact signing solution. It refers to the implementation of the Notation CLI and the Notation Go libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with 3rd party key management solutions through a plugin model. The CLI itself is referred to as the "Notation CLI," while the libraries are referrred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. These libraries can be used independently of the Notation CLI.
+
+
+- ### Notary Project Specifications
```
Comment (yizha1) [inline content/en/docs/glossary.md:14]: Remove `-`

File: content/en/docs/glossary.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+- [Notary Project OCI Signature Specification](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md)
+- [Notary Project OCI Signing and Verification Workflow](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-and-verification-workflow.md)
+- [Notary Project OCI COSE Envelope](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-cose.md)
+- [Notary Project OCI JWS Envelope](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-jws.md)
+
+**Signing Scheme Specification:** The Signing Scheme Specification is not specific to any particular storage method. It describes the supported signing schemes. Any tool that supports those schemes might be able to produce signatures that are compatible with Notary Project tools. The specification under this category includes the [Notary Project Signing Scheme.](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-scheme.md)
+
+
+- ### TUF
```
Comment (yizha1) [inline content/en/docs/glossary.md:29]: Remove `-`

**Review by yizha1 — COMMENTED** [review]
Thanks @Roseline-Bassey . A few comments

File: content/en/docs/glossary.md (hunk 1 above)
Comment (Roseline-Bassey) [reply content/en/docs/glossary.md]: resolve

File: content/en/docs/glossary.md (hunk 3 above)
Comment (Roseline-Bassey) [reply content/en/docs/glossary.md]: resolve

File: content/en/docs/glossary.md (hunk 7 above)
Comment (Roseline-Bassey) [reply content/en/docs/glossary.md]: resolve

File: content/en/docs/glossary.md (hunk 4 above)
Comment (Roseline-Bassey) [reply content/en/docs/glossary.md]: resolve

File: content/en/docs/glossary.md (hunk 2 above)
Comment (Roseline-Bassey) [reply content/en/docs/glossary.md]: resolve

**Comment by Roseline-Bassey** [comment]
Thanks, @yizha1 
I have uodated this PR to addressed your feedback

File: content/en/docs/glossary.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+---
+title: Glossary
+description: A list of Notary Project's terminologies 
+weight: 11
+---
+### Notary 
+Notary is a TUF [(The Update Framework)](https://github.com/theupdateframework/specification) based artifact signing solution. It consists of both [server](https://github.com/notaryproject/notary/blob/master/cmd/notary-server) and [client](https://github.com/notaryproject/notary/blob/master/cmd/notary) components for running and interacting with trusted collections. The client component is referred to as the Notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component.
```
Comment (yizha1) [inline content/en/docs/glossary.md:7]:
```suggestion
Notary is a TUF [(The Update Framework)](https://theupdateframework.com/) based artifact signing solution. It consists of both server and client components. The server infrastructure tightly integrates with the container registry and keeps track of the images pushed to the registry. The client component is referred to as the notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component. The most prominent use of this implementation is in Docker Content Trust (DCT). The server and the client implementation can be found in the [notary](https://github.com/notaryproject/notary) repository under the Notary Project organization.
```

File: content/en/docs/glossary.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+### Notation
+Notation is a non-TUF-based artifact signing solution. It refers to the implementation of the Notation CLI and the Notation Go libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with 3rd party key management solutions through a plugin model. The CLI itself is referred to as the "Notation CLI," while the libraries are referrred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. These libraries can be used independently of the Notation CLI.
```
Comment (yizha1) [inline content/en/docs/glossary.md:9]:
```suggestion
Notation is a non-TUF-based artifact signing solution. It refers to the implementation of the Notation CLI and the Notation libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with third party key management solutions through a plugin model. The CLI itself is referred to as the [Notation CLI](https://github.com/notaryproject/notation), while the libraries are referred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. These libraries can be used independently of the Notation CLI.
```

File: content/en/docs/glossary.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+### Notary Project Specifications
+Notary Project Specifications are a collection of specifications distributed across "subprojects" within the Notary Project. These specifications are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary project tooling.
```
Comment (yizha1) [inline content/en/docs/glossary.md:11]:
```suggestion
Notary Project Specifications are a collection of specifications distributed across "subprojects" within the Notary Project. These specifications are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary project tooling. These specifications are stored in the [notaryproject](https://github.com/notaryproject/notaryproject) repository under the Notary Project.
```

File: content/en/docs/glossary.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+
+Notary Project specifications are categorized into:
+
+**OCI Signature Specifications:**
+These specifications are specific to the Open Container Initiative (OCI). It describes how the Notary Project's Signing Scheme is applied to signatures stored in OCI registries. This category includes the following specifications:
```
Comment (yizha1) [inline content/en/docs/glossary.md:16]:
```suggestion
These specifications are specific to the [Open Container Initiative (OCI)](https://github.com/opencontainers). It describes how the Notary Project's Signing Scheme is applied to signatures stored in OCI registries. This category includes the following specifications:
```

File: content/en/docs/glossary.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+- [Notary Project OCI COSE Envelope](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-cose.md)
+- [Notary Project OCI JWS Envelope](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-jws.md)
+
+**Signing Scheme Specification:** The Signing Scheme Specification is not specific to any particular storage method. It describes the supported signing schemes. Any tool that supports those schemes might be able to produce signatures that are compatible with Notary Project tools. The specification under this category includes the [Notary Project Signing Scheme.](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-scheme.md)
+### TUF
+The Update Framework(TUF) is a project under the Notary Project that aims to implement the full TUF specification in a registry-native way. The initial version of Notary did not support transferring signatures between different registries and did not gain widespread adoption. The TUF project aims to develop an enhanced version of Notary, addressing these limitations and encouraging widespread adoption and usage.
```
Comment (yizha1) [inline content/en/docs/glossary.md:24]:
```suggestion
TUF refers to [The Update Framework[(https://theupdateframework.com/). A [tuf](https://github.com/notaryproject/tuf) repository under the Notary Project organization is intended for prototyping the storage of TUF metadata in OCI-compliant registries. It is not under active development at the moment.
```

**Review by yizha1 — COMMENTED** [review]
A few comments left

File: content/en/docs/glossary.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+### Notary Project Specifications
+Notary Project Specifications are a collection of specifications distributed across "subprojects" within the Notary Project. These specifications are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary project tooling.
+
+Notary Project specifications are categorized into:
+
+**OCI Signature Specifications:**
+These specifications are specific to the Open Container Initiative (OCI). It describes how the Notary Project's Signing Scheme is applied to signatures stored in OCI registries. This category includes the following specifications:
+- [Notary Project OCI Signature Specification](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md)
```
Comment (FeynmanZhou) [inline content/en/docs/glossary.md:17]: The `main` branch is a development branch so the content may be changed anytime. These URLs should be pointed to a specific tag (i.e. v1.0.0) after the `notaryproject` released v1.0.0 as described in https://github.com/notaryproject/notaryproject/issues/261.

File: content/en/docs/glossary.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+### Notation
+Notation is a non-TUF-based artifact signing solution. It refers to the implementation of the Notation CLI and the Notation Go libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with 3rd party key management solutions through a plugin model. The CLI itself is referred to as the "Notation CLI," while the libraries are referrred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. These libraries can be used independently of the Notation CLI.
+### Notary Project Specifications
```
Comment (FeynmanZhou) [inline content/en/docs/glossary.md:10]: I think we need to explain a dedicated term `Notary Project` in the page.

File: content/en/docs/glossary.md (hunk 13 above)
Comment (Roseline-Bassey) [reply content/en/docs/glossary.md:17]: Acknowledged, I'll ensure to point the URLs to the specific tag (v1.0.0) after the `notaryproject` releases  v1.0.0.

File: content/en/docs/glossary.md (hunk 14 above)
Comment (Roseline-Bassey) [reply content/en/docs/glossary.md:14]: I have included the Notary Project term in the glossary and provided an explanation for it.

File: content/en/docs/glossary.md [hunk 15]
```diff
@@ -0,0 +1,28 @@
+---
+title: Glossary
+description: A list of Notary Project's terminologies 
+weight: 11
+---
+### Notary 
+Notary is a TUF [(The Update Framework)](https://theupdateframework/) based artifact signing solution. It consists of both server and client components. The server infrastructure tightly integrates with the container registry and keeps track of the images pushed to the registry. The client component is referred to as the notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component. The most prominent use of this implementation is in Docker Content Trust (DCT). The server and the client implementation can be found in the [notary](https://github.com/notaryproject/notary) repository under the Notary Project organization.
```
Comment (iamsamirzon) [inline content/en/docs/glossary.md:7]:
```suggestion
Notary is a TUF [(The Update Framework)](https://theupdateframework/) based artifact signing **implementation**. It consists of a server and client components. The server infrastructure tightly integrates with the container registry and keeps track of the images pushed to the registry. The client component is referred to as the notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component. The most prominent use of this implementation is in Docker Content Trust (DCT). The server and the client implementation can be found in the [notary](https://github.com/notaryproject/notary) repository under the Notary Project organization.
```

File: content/en/docs/glossary.md [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+### Notation
+Notation is a non-TUF-based artifact signing solution. It refers to the implementation of the Notation CLI and the Notation libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with third-party key management solutions through a plugin model. The CLI itself is referred to as the [Notation CLI](https://github.com/notaryproject/notation) while the libraries are referred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. These libraries can be used independently of the Notation CLI.
```
Comment (iamsamirzon) [inline content/en/docs/glossary.md:9]:
```suggestion
Notation, based on new set of specifications unrelated to TUF, is a newer implementation from the Notary Project. It refers to the Notation CLI and the Notation libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with third-party key management solutions through a plugin model. The CLI itself is referred to as the [Notation CLI](https://github.com/notaryproject/notation) while the libraries are referred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. Stakeholders can use libraries independently of the Notation CLI for integrating Notation with their own client tools.
```

File: content/en/docs/glossary.md [hunk 17]
```diff
@@ -0,0 +1,28 @@
...
+
+### Notary Project 
+Notary Project refers to the [GitHub organization](https://github.com/notaryproject), the community, all specifications, and all the repositories under the organization, including community-released tools like Notary and Notation.
```
Comment (iamsamirzon) [inline content/en/docs/glossary.md:12]:
```suggestion
The Notary Project refers to the [GitHub organization](https://github.com/notaryproject), the community, all specifications, and all the repositories under the GitHub organization, including community-released tools like Notary and Notation.
```

File: content/en/docs/glossary.md [hunk 18]
```diff
@@ -0,0 +1,28 @@
...
+
+### Notary Project Specifications
+Notary Project Specifications are a collection of specifications distributed across "subprojects" within the Notary Project. These specifications are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary project tooling. These specifications are stored in the [notaryproject](https://github.com/notaryproject/notaryproject) repository under the Notary Project.
```
Comment (iamsamirzon) [inline content/en/docs/glossary.md:15]:
```suggestion
Notary Project Specifications, mainly created between years 2021-2022, are a collection of specifications distributed across "subprojects" within the Notary Project. Notation CLI and libraries have implemented these specifications, which are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary project tooling. These specifications are stored in the [notaryproject](https://github.com/notaryproject/notaryproject) repository under the Notary Project.
```

**Review by iamsamirzon — CHANGES_REQUESTED** [review]

File: content/en/docs/glossary.md [hunk 19]
```diff
@@ -0,0 +1,28 @@
...
+### Notation
+Notation, based on new set of specifications unrelated to TUF, is a newer implementation from the Notary Project. It refers to the implementation of the Notation CLI and the Notation libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with third-party key management solutions through a plugin model. The CLI itself is referred to as the [Notation CLI](https://github.com/notaryproject/notation) while the libraries are referred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. Stakeholders can use libraries independently of the Notation CLI for integrating Notation with their own client tools.
+
+### Notary Project
```
Comment (gokarnm) [inline content/en/docs/glossary.md:11]: Suggest moving Notary Project as the first glossary entry to set overall context and as it is referenced in Notation enrty.

**Review by gokarnm — DISMISSED** [review]
LGTM!

File: content/en/docs/glossary.md [hunk 20]
```diff
@@ -0,0 +1,26 @@
+---
+title: Glossary
+description: A list of Notary Project's terminologies 
+weight: 11
+---
+### Notary Project
```
Comment (zr-msft) [inline content/en/docs/glossary.md:6]:
```suggestion
## Notary Project 
```

File: content/en/docs/glossary.md [hunk 21]
```diff
@@ -0,0 +1,26 @@
+---
+title: Glossary
+description: A list of Notary Project's terminologies 
+weight: 11
+---
+### Notary Project 
+The Notary Project refers to the [GitHub organization](https://github.com/notaryproject), the community, all specifications, and all the repositories under the GitHub organization, including community-released tools like Notary and Notation.
+### Notary
```
Comment (zr-msft) [inline content/en/docs/glossary.md:8]:
```suggestion
## Notary 
```

File: content/en/docs/glossary.md [hunk 22]
```diff
@@ -0,0 +1,26 @@
+---
+title: Glossary
+description: A list of Notary Project's terminologies 
+weight: 11
+---
+### Notary Project 
+The Notary Project refers to the [GitHub organization](https://github.com/notaryproject), the community, all specifications, and all the repositories under the GitHub organization, including community-released tools like Notary and Notation.
+### Notary 
+Notary is a TUF [(The Update Framework)](https://theupdateframework/) based artifact signing **implementation**. It consists of a server and client components. The server infrastructure tightly integrates with the container registry and keeps track of the images pushed to the registry. The client component is referred to as the notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component. The most prominent use of this implementation is in Docker Content Trust (DCT). The server and the client implementation can be found in the [notary](https://github.com/notaryproject/notary) repository under the Notary Project organization.
```
Comment (zr-msft) [inline content/en/docs/glossary.md:9]:
```suggestion
Notary is a TUF [(The Update Framework)](https://theupdateframework.com/) based artifact signing **implementation**. It consists of a server and client components. The server infrastructure tightly integrates with the container registry and keeps track of the images pushed to the registry. The client component is referred to as the notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component. The most prominent use of this implementation is in Docker Content Trust (DCT). 
```

File: content/en/docs/glossary.md [hunk 23]
```diff
@@ -0,0 +1,26 @@
...
+### Notation
```
Comment (zr-msft) [inline content/en/docs/glossary.md:10]:
```suggestion
## Notation
```

File: content/en/docs/glossary.md [hunk 24]
```diff
@@ -0,0 +1,26 @@
...
+Notation, based on new set of specifications unrelated to TUF, is a newer implementation from the Notary Project. It refers to the implementation of the Notation CLI and the Notation libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with third-party key management solutions through a plugin model. The CLI itself is referred to as the [Notation CLI](https://github.com/notaryproject/notation) while the libraries are referred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. Stakeholders can use libraries independently of the Notation CLI for integrating Notation with their own client tools.
+### Notary Project Specifications
```
Comment (zr-msft) [inline content/en/docs/glossary.md:12]:
```suggestion
## Notary Project Specifications
```

File: content/en/docs/glossary.md [hunk 25]
```diff
@@ -0,0 +1,26 @@
...
+- [Notary Project OCI Signature Specification](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md)
+- [Notary Project OCI Signing and Verification Workflow](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-and-verification-workflow.md)
+- [Notary Project OCI COSE Envelope](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-cose.md)
+- [Notary Project OCI JWS Envelope](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-jws.md)
+
+**Signing Scheme Specification:** The Signing Scheme Specification is not specific to any particular storage method. It describes the supported signing schemes. Any tool that supports those schemes might be able to produce signatures that are compatible with Notary Project tools. The specification under this category includes the [Notary Project Signing Scheme.](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-scheme.md)
+### TUF
```
Comment (zr-msft) [inline content/en/docs/glossary.md:25]:
```suggestion
## TUF
```

**Review by zr-msft — CHANGES_REQUESTED** [review]

**Review by zr-msft — APPROVED** [review]

File: content/en/docs/glossary.md [hunk 26]
```diff
@@ -0,0 +1,26 @@
...
+## Notary Project Specifications
+Notary Project Specifications, mainly created between years 2021-2022, are a collection of specifications distributed across "subprojects" within the Notary Project. Notation CLI and libraries have implemented these specifications, which  are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary project tooling. These specifications are stored in the [notaryproject](https://github.com/notaryproject/notaryproject) repository under the Notary Project.
+
+Notary Project specifications are categorized into:
+
+**OCI Signature Specifications:**
+These specifications are specific to the [Open Container Initiative (OCI)](https://github.com/opencontainers). It describes how the Notary Project's Signing Scheme is applied to signatures stored in OCI registries. This category includes the following specifications:
+- [Notary Project OCI Signature Specification](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md)
```
Comment (sajayantony) [inline content/en/docs/glossary.md:19]: This repo has been renamed to specifications.

Comment (Roseline-Bassey) [reply content/en/docs/glossary.md]: Thanks for the update @sajayantony 
The URLs are updated.

File: content/en/docs/glossary.md [hunk 27]
```diff
@@ -0,0 +1,26 @@
+---
+title: Glossary
+description: A list of Notary Project's terminologies 
+weight: 11
+---
+## Notary Project 
+The Notary Project refers to the [GitHub organization](https://github.com/notaryproject), the community, all specifications, and all the repositories under the GitHub organization, including community-released tools like Notary and Notation.
```
Comment (yizha1) [inline content/en/docs/glossary.md:7]:
```suggestion
The Notary Project refers to the [GitHub organization](https://github.com/notaryproject), the community, all specifications, and all the repositories under the GitHub organization, including community-released tools like [notary](https://github.com/notaryproject/notary) and [notation](https://github.com/notaryproject/notation).
```

File: content/en/docs/glossary.md [hunk 28]
```diff
@@ -0,0 +1,26 @@
...
+## Notary Project Specifications
+Notary Project Specifications, mainly created between years 2021-2022, are a collection of specifications distributed across "subprojects" within the Notary Project. Notation CLI and libraries have implemented these specifications, which  are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary project tooling. These specifications are stored in the [notaryproject](https://github.com/notaryproject/specifications) repository under the Notary Project.
```
Comment (yizha1) [inline content/en/docs/glossary.md:13]:
```suggestion
Notary Project Specifications, mainly created between years 2021-2022, are a collection of specifications distributed across "subprojects" within the Notary Project. Notation CLI and libraries have implemented these specifications, which are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary Project tooling. These specifications are stored in the [specifications](https://github.com/notaryproject/specifications) repository under the Notary Project.
```

**Review by yizha1 — COMMENTED** [review]
Thanks @Roseline-Bassey. Left some comments.

**Review by gokarnm — APPROVED** [review]
LGTM!

**Comment by iamsamirzon** [comment]
LGTM

**Comment by yizha1** [comment]
> LGTM

@iamsamirzon Thanks for approving this PR, but it seems you need to remove "changed request", otherwise this PR cannot be merged.

**Review by yizha1 — APPROVED** [review]
LGTM

**Review by iamsamirzon — APPROVED** [review]

**Comment by iamsamirzon** [comment]
LGTM

**Pull request merged**